### ISSUE TITLE
no release notes

### ISSUE THREAD

**Issue by badescuga** [creditkarma/thrift-typescript#127]
I'd be very helpful if you'd provide release notes. This would help with fixing bugs and understanding why you should/shouldn't move between versions (like from 1.x to 2.x)

**Comment by kevin-greene-ck** [comment]
Agreed, will get this going for future releases and include notes about 2.x v 1.x.
